<header class="banner"><div class="banner">
  <div class="logos">
    <div><img title="Julia logo" src="../figures/julia-logo-color.svg" alt="Julia animated logo" width=35 /></div>
    <div>◌</div>
    <div><img title="Powered by MathJax" src="https://www.mathjax.org/badge/mj-logo.svg"
              alt="Powered by MathJax" width=65 />
    </div>
  </div>  <!-- logos -->
  <div class="cite_textbook ms9">1-023 <span class="authors"></span></div>
</div></header>


<footer class="navbar navbar-default navbar-fixed-bottom mcs-psy-nav" >
  <div class="container-fluid text-right h4">
    <em>ψᵟ</em>
  </div>
</footer>

A closed system consisting of 2 lb of a gas undergoes a process during which the relation between pressure and volume is expressed as $PV^n = \small {\rm constant}$. The process begins with $P_1 = 25\, {\small \rm lbf/in}^2$, $V_1 = 10\, \small {\rm ft^3}$ and ends with $P_2 = 100\, \small {\rm lbf/in^2}$ The value of $n = 1.3.$ Determine the final volume, $V_2$, in $\small{\rm ft^3}$, and determine the specific volume at states 1 and 2, in $\small {\rm ft^3/lbm}$.

<div class="btn-info">&nbsp;<div class="badge"></div><i class="fa fa-question"></i><strong> Sidebar: Thermo notes</strong> </div>
<div class="panel-info"><div class="panel-heading">
    

Processes where $\small PV^n = \textsf{constant}$ are called **polytropic processes**. They are very common; some examples are piston-cylinder devices like bicycle air pumps, engines, lungs, etc. You'll see them soon in Chapter 2.
</div></div>

### Given   

System undergoing polytropic process: $PV^n = $ constant   

$\quad
\begin{equation*}
\left.\begin{split}
P_1 &= 25\, \rm psi\\
V_1 &= 10\,\rm ft^3
\end{split} \right\}
\xrightarrow[2\,\rm lbm]{\text{polytropic, }PV^n = \textbf{ K},~n=1.3 }\left\{\begin{split}
P_1 &= 100\, \rm psi\\
V_1 &= \text{?}\,\rm ft^3
\end{split}
\right.
\end{equation*}
$


### Find  

1. ${V_2}$ ft³ 
1. $v_1$ ft³/lbm  
1. $v_2$ ft³/lbm


### Assumptions  

There is no mention of gage pressure anywhere, so assume the pressures you are given are absolute pressures.  

<br>
<div class="btn-warning">&nbsp;<div class="badge"></div><i class="fa fa-exclamation-circle"></i><strong> Pro Tip #354:</strong></div>
<div class="panel-warning"><div class="panel-heading">   
   $\small \textit{Always work using absolute pressures, and convert your output to gage pressure when necessary.}$</div></div>

## Attack   

We're organized and calm and ready to find everything we're supposed to find.  

### Get Organized

You must get organized before you start wildly writing equations and numbers. 
Generally, consider what you need to find, and mentally plot a path to get there from what you know.  

We are given a process in which  

$$PV^n = \small \textit{constant}$$  

which translates to  

$$P_1{V_1}^n = P_2{V_2}^n$$    

We need $V_2$. Conveniently, 

$$V_2 = V_1 \left(\frac{P_1}{P_2}\right)^{1/n} \tag{1} \label{eq:V_2} \\[18pt]$$   

From here, we calculate $v_1$ and $v_2$ directly from the definition of specific volume  

$$v \equiv \frac{V}{m}. \tag{2} \label{eq:v}$$  

In [1]:
# given

P₁_psi = 25   # psi
V₁_ft³ = 10   # ft³
P₂_psi = 100  # psi
m_lbm = 2     # lbm
n = 1.3;      # unitless

V₂_ft³ = V₁_ft³ * (P₁_psi/P₂_psi)^(1/n)   # from eq(1)

𝑣₁_ft³lbm⁻¹ = V₁_ft³ / m_lbm   # from eq(2)
𝑣₂_ft³lbm⁻¹ = V₂_ft³ / m_lbm   # from eq(2)


@show V₂_ft³ 𝑣₁_ft³lbm⁻¹ 𝑣₂_ft³lbm⁻¹  ;

V₂_ft³ = 3.4425236279856737
𝑣₁_ft³lbm⁻¹ = 5.0
𝑣₂_ft³lbm⁻¹ = 1.7212618139928368


<div class="btn-success">&nbsp;<div class="badge"></div><i class="fa fa-bomb"></i><strong> BAM</strong></div>
<div class="panel-success"><div class="panel-heading">   

$$ 
V_2 = 3.443\,{\rm ft^3}           \qquad 
v_1 = 5    \,{\rm\frac{ft^3}{lb}} \qquad
v_2 = 1.721\,{\rm\frac{ft^3}{lb}} 
$$
  
</div></div>

<footer class="psy"><div class="psy">ψᵟ</div></footer>